builds/service-onboarding-build-pack/Jenkinsfile

```diff
@@ -141,11 +142,17 @@ node  {
 
 	stage('Get Service Template'){
 		try {
+			try {
+				sh 'rm -rf *'
```

Comment:
`rm -rf *` should just do it. You don't need the next cmd.

Reply:
Done